examples/example_03_base_workflow.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+"""Run simple DFT calculation"""
+
+from __future__ import print_function
+from __future__ import absolute_import
+
+import sys
+import click
+
+from aiida.engine import run, run_get_node
+from aiida.orm import Code, Dict, StructureData
+from aiida.common import NotExistent
+from aiida.plugins import WorkflowFactory
+
+import pymatgen as mg
+
+GaussianBaseWorkChain = WorkflowFactory('gaussian.base')
+
+
+def example_dft(gaussian_code):
```

Comment:
This example produces the Asytop error, which is correctly handled, right? maybe add a small comment.

Reply:
ah sorry, it's already there!

Reply:
But the comment seems to be unfinished

Reply:
Yes, sorry, not sure how did this happen. Fixed.